README.md

```diff
@@ -56,80 +63,71 @@ There are four kinds of Gauss-Chebyshev quadrature rules, corresponding to four
 
 4. 4th kind, weight function `w(x) = sqrt((1-x)/(1+x))`
 
-They are all have explicit simple formulas for the nodes and weights <a href="http://books.google.com/books?id=8FHf0P3to0UC&lpg=PP1&pg=PA180#v=onepage&q&f=false">[4]</a>.
+They are all have explicit simple formulas for the nodes and weights [[4]](https://books.google.co.jp/books?id=8FHf0P3to0UC).
+
 ## The algorithm for Gauss-Legendre
 Gauss quadrature for the weight function `w(x) = 1`.
 
 * For `n<=5`: Use an analytic expression.
-
 * For `n<=60`: Use Newton's method to solve `Pn(x)=0`. Evaluate `Pn` and `Pn'` by 3-term recurrence. Weights are related to `Pn'`.
-
-* For `n>60`: Use asymptotic expansions for the Legendre nodes and weights <a href="http://epubs.siam.org/doi/abs/10.1137/140954969">[1]</a>.  
+* For `n>60`: Use asymptotic expansions for the Legendre nodes and weights [[1]](http://epubs.siam.org/doi/abs/10.1137/140954969).
 
 ## The algorithm for Gauss-Jacobi
 Gauss quadrature for the weight functions `w(x) = (1-x)^a(1+x)^b`, `a,b>-1`.
 
 *  For `n<=100`: Use Newton's method to solve `Pn(x)=0`. Evaluate `Pn` and `Pn'` by three-term recurrence.
-
-*  For `n>100`: Use Newton's method to solve `Pn(x)=0`. Evaluate `Pn` and `Pn'` by an asymptotic expansion (in the interior of `[-1,1]`) and the three-term recurrence `O(n^-2)` close to the endpoints. (This is a small modification to the algorithm described in <a href="http://epubs.siam.org/doi/abs/10.1137/120889873">[3]</a>.)
-
+*  For `n>100`: Use Newton's method to solve `Pn(x)=0`. Evaluate `Pn` and `Pn'` by an asymptotic expansion (in the interior of `[-1,1]`) and the three-term recurrence `O(n^-2)` close to the endpoints. (This is a small modification to the algorithm described in [[3]](http://epubs.siam.org/doi/abs/10.1137/120889873).)
 * For `max(a,b)>5`: Use the Golub-Welsch algorithm requiring `O(n^2)` operations. 
 
 ## The algorithm for Gauss-Radau
 Gauss quadrature for the weight function `w(x)=1`, except the endpoint `-1` is included as a quadrature node.
 
-The Gauss-Radau nodes and weights can be computed via the `(0,1)` Gauss-Jacobi nodes and weights <a href="http://epubs.siam.org/doi/abs/10.1137/120889873">[3]</a>.
+The Gauss-Radau nodes and weights can be computed via the `(0,1)` Gauss-Jacobi nodes and weights[[3]]("http://epubs.siam.org/doi/abs/10.1137/12088987)>.
```

Comment:
Is the trailing `>` here a typo leftover?

Reply:
Thanks. I'll fix it.